### ISSUE TITLE
Recording Audio

### ISSUE THREAD

**Issue by nomatica** [CarnetApp/CarnetElectron#42]
After installing the app in nextcloud, I tried to record through the browser.  
Browser asks for access to mic.  
Then  the audio meter changes as though it is recording. But When I press the button to stop the timer keeps going. Any ideas how to address this?
Thank you. Great app

**Comment by PhieF** [comment]
Hm it doesn't happen here, is your domain in https ? With which web browser ?

Thanks for the report!

**Comment by nomatica** [comment]
yes connected via https.   I tried using  firefox initially then  windows chrome edge. I am not sure if I attempted with google chrome.
